lib/themes/dosomething/paraneue_dosomething/js/takeover/Takeover.js

```diff
@@ -161,6 +109,78 @@ class Takeover {
   isActive($element) {
     return $element.hasClass(this.activeClass);
   }
+
+  /**
+   * Adds validation message to an element
+   *
+   * @param  {jQuery}   $element
+   * @param  {string}   msg
+   */
+  showValidationMessage($element, msg) {
+    Validation.showValidationMessage($element, {message: msg});
+  }
+
+  /**
+   * Prepare fields for validation.
+   *
+   * @param  {jQuery}   $fields
+   */
+  prepareFieldValidation($fields) {
+    Validation.prepareFields($fields);
+  }
+
+  /*
+   * Handles back buttons.
+   */
+  goBack() {
+    const $backButton = this.$el.find('.takeover__screen .back-button');
+
+    if ($backButton.length) {
+      $backButton.click(e => {
+        e.preventDefault();
+
+        this.changeScreens('previous');
+      });
+    }
+  }
+
+  /*
+   * Load a specific screen in the modal flow.
+   *
+   * @param  {jQuery}   $screen
+   */
+  loadScreen($screen) {
+    this.$el.find(`.takeover__screen.${this.activeClass}`).removeClass(this.activeClass);
+    $screen.addClass(this.activeClass);
+  }
+
+  /*
+   * Add a compiled lodash template to a container element.
+   *
+   * @param  {jQuery}   $container
+   * @param  {string}   tpl
+   * @param  {object}   data
+   */
+  addTemplate($container, tpl, data) {
+    data = (data) ? data : {};
+
+    const markup = template(tpl);
+
+    $container.html(markup(data))
+  }
+
+  /*
+   * Resize an elemen based on the height of it's contents
+   *
+   * @param  {jQuery}   $container
+   * @param  {jQuery}   $referenceHeight
+   * @param  {int}      padding
+   */
+  resizeContainerHeight($container, $referenceHeight, padding = null) {
+    padding = (padding === null) ? 100 : padding;
```

Comment:
Can you use `100` as the default `padding` parameter here rather than `null` and the ternary?